packages/react/src/editor/BlockNoteView.tsx

```diff
@@ -174,11 +174,21 @@ function BlockNoteViewComponent<
           defaultUIProps,
         }}>
         <ElementRenderer ref={setElementRenderer} />
-        {renderEditor ? (
-          <BlockNoteViewEditor ref={ref}>{children}</BlockNoteViewEditor>
-        ) : (
-          children
-        )}
+        <div
```

Comment:
Nice improvement, I think this could work. We need to check all references to `bn-container` in the codebase to check whether this is safe (both in CSS and TS)

Let's also create a separate component `BlockNoteContainer` for this div